x-pack/plugins/security_solution/public/network/components/embeddables/map_tool_tip/map_tool_tip.tsx

```diff
@@ -134,7 +134,7 @@ export const MapToolTipComponent = ({
         try {
           const featureGeo = loadFeatureGeometry({ layerId, featureId });
           const [featureProperties, layerNameString] = await Promise.all([
-            loadFeatureProperties({ layerId, featureId, mbProperties: {} }),
+            loadFeatureProperties({ layerId, featureId }),
```

Comment:
I have been investigating this further and I think instead of removing mbProperties, you should pass in `mbProperties: features[featureIndex].mbProperties`. Then you do not need to change the interface in Maps.